### ISSUE TITLE
Warnings for type beaker: is bigger on the inside.

### ISSUE THREAD

**Issue by hexagonrecursion** [CleverRaven/Cataclysm-DDA#42274]
I get the following message when loading a world
```
 DEBUG    : warnings for type beaker:
is bigger on the inside.  consider using TARDIS flag.


 FUNCTION : void Item_factory::check_definitions() const
 FILE     : src/item_factory.cpp
 LINE     : 1422
```

# Steps To Reproduce

1. Create a new world
2. Open the "Custom Character" menu
3. Select the new world
4. See the above asseertion

# Expected behavior

No assertions are triggered.

# Screenshots

none

# Versions and configuration

```
- OS: Linux
    - OS Version: LSB Version: :core-4.1-amd64:core-4.1-noarch; Distributor ID: Fedora; Description: Fedora release 32 (Thirty Two); Release: 32; Codename: ThirtyTwo; 
- Game Version: 0.E-4254-gfea0266 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food]
]
```

# Additional context

https://github.com/CleverRaven/Cataclysm-DDA/blob/fea0266d6eaa9b49486aa6efdda30fc95ac10d2d/data/json/items/containers.json#L923-L950

[blame](https://github.com/CleverRaven/Cataclysm-DDA/blame/fea0266d6eaa9b49486aa6efdda30fc95ac10d2d/data/json/items/containers.json#L923-L950)

Most likely introduced by 792f0562c6a9be66e45f8dd28f0c4111182e6733, #42213

**Comment by matt32106** [comment]
Same here

**Comment by Hulkninja** [comment]
Me too. Are you guys getting the warning message on new worlds/characters too, or just on old ones made before the update..?

**Comment by Taberone** [comment]
Also getting this

**Comment by anothersimulacrum** [comment]
Fixed 12e8c9b84945abcee1a5a1a45e966a243935d695
